### PR TITLE
[docs] added python example

### PR DESCRIPTION
**Pull request by svenefftinge**
Adds a section with a python example.

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/583"></a>

### CODE REVIEW

File: src/routes/docs/quickstart/python.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+title: Python example
+---
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Python Django Quickstart
+
+Learn how to set-up and understand the benefits of Gitpod **in less than 5 min** with our [Go](https://github.com/gitpod-io/example-python-django) example.
```
Comment (mikenikles) [inline src/routes/docs/quickstart/python.md:12]: That should be "Python" as the link text.

**Review by mikenikles — APPROVED** [review]
One typo, otherwise ready to go. I'm on mobile, otherwise I'd have made the change and merged.

File: src/routes/docs/quickstart/python.md (hunk 1 above)
Comment (nisarhassan12) [reply src/routes/docs/quickstart/python.md:12]: I have addressed this in https://github.com/gitpod-io/website/pull/589

Comment (nisarhassan12) [reply src/routes/docs/quickstart/python.md:12]: I have closed my Pr in favour of https://github.com/gitpod-io/website/pull/588 from Johannes which addresses this.

**Pull request merged**